### PR TITLE
Mocking up Section 5

### PR DESCRIPTION
**Pull request by CalvinRodo**
Please note there is no validation on these pages, and most of it is
copy and pasted to be as true to the paper form as possible.

### CODE REVIEW

File: .vscode/node-starter-app.code-snippets [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
```
Comment (jeffmaher) [inline .vscode/node-starter-app.code-snippets]: My fault when setting up the repo! Could you add `.vscode` to the `.gitignore` and/or remove this file?

**Review by jeffmaher — CHANGES_REQUESTED** [review]
Looks good to me! Just one nitpick on a VSCode file slipping in.

File: .vscode/node-starter-app.code-snippets (hunk 1 above)
Comment (jeffmaher) [reply .vscode/node-starter-app.code-snippets]: Found myself with some files too, so did this: https://github.com/cds-snc/cppd-medical-report/pull/24

Comment (CalvinRodo) [reply .vscode/node-starter-app.code-snippets]: I'll pull in the change you merged already and remove the files. 

I was adding some snippets for the macros, it's still a bit fiddly so probably doesn't need to be shared.

**Review by jeffmaher — DISMISSED** [review]
LGTM! 🍕

**Review by jeffmaher — APPROVED** [review]

**Pull request merged**